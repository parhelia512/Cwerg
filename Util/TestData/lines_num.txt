int64 -1000
int64 -9223372036854775808
int64 0x8000000000000000
int64 0xffffffffffffffff
int64 4.5
int64 +nan
int64 0.0
int64 -0.0
uint64 1000
uint64 9223372036854775808
uint64 0x8000000000000000
uint64 0xffffffffffffffff
uint64 4.5
uint64 +nan
uint64 0.0
uint64 -0.0
flt64 4.5
flt64 0.25
flt64 nan
flt64 +nan
flt64 -inf
flt64 -INF
flt64 .0
flt64 1e4
flt64 +inf
flt64 0.0
flt64 -0.0
flt64 0x7ff8000000000000
flt64 0x0
flt64 0x3fd0000000000000
flt64 0.25
flt64 .0009765625
char 'a'
char '\n'
char '\x77'
char '\x778'
