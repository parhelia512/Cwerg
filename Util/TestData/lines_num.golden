
int64 -1000
[INT64] -1000 fffffffffffffc18

int64 -9223372036854775808
[INT64] -9223372036854775808 8000000000000000

int64 0x8000000000000000
[INT64] -9223372036854775808 8000000000000000

int64 0xffffffffffffffff
[INT64] -1 ffffffffffffffff

int64 4.5
[INT64] @BAD VALUE@

int64 +nan
[INT64] @BAD VALUE@

int64 0.0
[INT64] @BAD VALUE@

int64 -0.0
[INT64] @BAD VALUE@

uint64 1000
[UINT64] 1000 3e8

uint64 9223372036854775808
[UINT64] 9223372036854775808 8000000000000000

uint64 0x8000000000000000
[UINT64] 9223372036854775808 8000000000000000

uint64 0xffffffffffffffff
[UINT64] 18446744073709551615 ffffffffffffffff

uint64 4.5
[UINT64] @BAD VALUE@

uint64 +nan
[UINT64] @BAD VALUE@

uint64 0.0
[UINT64] @BAD VALUE@

uint64 -0.0
[UINT64] @BAD VALUE@

flt64 4.5
[FLT64] 4.5 4012000000000000

flt64 0.25
[FLT64] 0.25 3fd0000000000000

flt64 nan
[FLT64] nan 7ff8000000000000

flt64 +nan
[FLT64] nan 7ff8000000000000

flt64 -inf
[FLT64] -inf fff0000000000000

flt64 -INF
[FLT64] -inf fff0000000000000

flt64 .0
[FLT64] 0 0

flt64 1e4
[FLT64] 10000 40c3880000000000

flt64 +inf
[FLT64] inf 7ff0000000000000

flt64 0.0
[FLT64] 0 0

flt64 -0.0
[FLT64] -0 8000000000000000

flt64 0x7ff8000000000000
[FLT64] nan 7ff8000000000000

flt64 0x0
[FLT64] 0 0

flt64 0x3fd0000000000000
[FLT64] 0.25 3fd0000000000000

flt64 0.25
[FLT64] 0.25 3fd0000000000000

flt64 .0009765625
[FLT64] 0.000976562 3f50000000000000

char 'a'
[CHAR] 97

char '\n'
[CHAR] 10

char '\x77'
[CHAR] 119

char '\x778'
[CHAR] @BAD VALUE@
